TestLongData fails do to changed MySQL server on Travis

### Issue description
An old test started to fail on Travis. It seems like we need to adjust `innodb_log_file_size`.

### Error log
```
--- FAIL: TestLongData (0.37s)

	driver_test.go:155: error on exec [query too large to print]: Error 1118: The size of BLOB/TEXT data inserted in one transaction is greater than 10% of redo log size. Increase the redo log size using innodb_log_file_size.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,6 +5,7 @@ go:
   - 1.6
   - 1.7
   - 1.8
+  - 1.9
   - tip
 
 before_install:
@@ -16,7 +17,7 @@ matrix:
     - env: DB=MYSQL57
       sudo: required
       dist: trusty
-      go: 1.8
+      go: 1.9
       services:
         - docker
       before_install:
@@ -38,7 +39,7 @@ matrix:
     - env: DB=MARIA55
       sudo: required
       dist: trusty
-      go: 1.8
+      go: 1.9
       services:
         - docker
       before_install:
@@ -60,7 +61,7 @@ matrix:
     - env: DB=MARIA10_1
       sudo: required
       dist: trusty
-      go: 1.8
+      go: 1.9
       services:
         - docker
       before_install:

--- a/.travis.yml
+++ b/.travis.yml
@@ -12,6 +12,12 @@ before_install:
   - go get golang.org/x/tools/cmd/cover
   - go get github.com/mattn/goveralls
 
+before_script:
+  - echo -e "[server]\ninnodb_log_file_size=256MB\ninnodb_buffer_pool_size=512MB\nmax_allowed_packet=16MB" | sudo tee -a /etc/mysql/my.cnf
+  - sudo service mysql restart
+  - .travis/wait_mysql.sh
+  - mysql -e 'create database gotest;'
+
 matrix:
   include:
     - env: DB=MYSQL57
@@ -80,9 +86,6 @@ matrix:
         - export MYSQL_TEST_ADDR=127.0.0.1:3307
         - export MYSQL_TEST_CONCURRENT=1
 
-
-before_script:
-  - mysql -e 'create database gotest;'
 script:
   - go test -v -covermode=count -coverprofile=coverage.out
   - go vet ./...

